src/App.css

```diff
@@ -70,6 +90,51 @@
   width: 300%;
 }
 
+/* sets appearance and position of background when modal is open */
+.modal-container {
+  position: fixed;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
```

Comment:
Another way to do this that's not sensitive to if you add padding to the element is to specify:

    bottom: 0;
    left: 0;
    right: 0;
    top: 0;

Same effect, but no chance of accidental scroll bars.